assets/js/hooks/session.js

```diff
@@ -1442,6 +1446,15 @@ const Session = {
   getElement(name) {
     return this.el.querySelector(`[data-el-${name}]`);
   },
+
+  jumpToLine(file, line) {
+    const [_filename, cellId] = file.split("#cell:");
+
+    this.setFocusedEl(cellId, { scroll: false, focusElement: true });
```

Comment:
```suggestion
    this.setFocusedEl(cellId, { scroll: false });
```

(that's the default)